test/electron.test.js

```diff
@@ -4,6 +4,6 @@ describe('electron', function() {
   it('respects ELECTRON_VERSION', function() {
     process.env.ELECTRON_VERSION = '1.2.3';
     assert.throws(function() { require('..'); },
-                  (/Cannot find module .*\/node-sqlite3\/lib\/binding\/electron-v1.2-[^-]+-x64\/node_sqlite3.node/));
+                  (/Cannot find module .*\/lib\/binding\/electron-v1.2-[^-]+-x64\/node_sqlite3.node/));
```

Comment:
Hmm, this now works for me when run as `./node_modules/.bin/mocha test/electron.test.js`, but fails with `npm test` with 
```
     AssertionError: Missing expected exception..
      at _throws (assert.js:346:5)
      at Function.assert.throws (assert.js:370:3)
      at Context.<anonymous> (test/electron.test.js:6:12)
```